addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2000,6 +2014,14 @@ options.registry.MobileVisibility = options.Class.extend({
         }
         return await this._super(...arguments);
     },
+
+    /**
+     * @override
+     */
+    async _computeVisibility() {
+        // Same as default but overridden by other apps
+        return true;
+    },
```

Comment:
Useless, was added before when there was nothing else in the class to make it less confusing. Rebase the branch, I merged a fix which already removes the extra MobileVisibility class.